README.md

```diff
@@ -1,5 +1,8 @@
 # Theme Converter for Visual Studio
- This is a CLI tool that takes in a VS Code theme’s json file and converts it into a VS theme. 
+Theme Converter is a CLI tool, which allows you to convert your VS Code theme to work in Visual Studio. It's a simple tool that maps the colors of a theme to Visual Studio. Themes can be turned into VSIXs and uploaded to the Marketplace so that a more people can use your theme! This allows developers who use Visual Studio, or use both VS Code and Visual Studio, to quickly get access to a wide range of fun and interesting themes! 
+
+Here's a quick overview on how the Theme Converter works:
```

Comment:
if the rich diff is accurate, the thumbnail appears on the right rather than below
maybe try to add a blank line in between lines 4 and 5